### ISSUE TITLE
Disable translations for specific query

### ISSUE THREAD

**Issue by rafailml** [dimsav/laravel-translatable#482]
Here is my case: 
I have product which have a categories. When I edit the product I don't want to get translated categories. I want only category.id
How can I achieve that?

Here is what I'm doing now:
```
$product = Product::select('id', 'sku',...)
->with(['categories' => function ($query) {
                        $query->select(['categories.id']);
                    }])
->where('id', $id)
->first();
```
But I'm getting categories with their translations. Is it possible for that query only to get categories without translations?

**Comment by Gummibeer** [comment]
Have you set `$with` property in the category class? otherwise the translations are only queried if you access a translated attribute.

**Comment by rafailml** [comment]
No, I didn't.
I have  only
`public $translatedAttributes = ['name'];`
in my Category model

**Comment by Gummibeer** [comment]
And in which situation you find-out/see that the translations are also loaded? After your query no translation should be available.

**Comment by rafailml** [comment]
In fact every query, even in other models returns translations. And I don't have `$with` anywhere.

**Comment by Gummibeer** [comment]
This package doesn't have any join - so the original query can't load translations. So what are you doing/your code that you see the translations? Do you use `toArray()` on the model? 
https://github.com/dimsav/laravel-translatable/blob/5f19c5b9253ea49baa935135ae2ffd37f81bc5d4/src/config/translatable.php#L108-L117

**Comment by Gummibeer** [comment]
And what's the code you execute where you see the translated fields?
It's impossible to tell you why you see translated fields if you don't show us what you do.

**Comment by rafailml** [comment]
Thats is the code:

```
public function simpleCategories()
    {
        $categories = Category::select('id')
            ->get();
       return response()->json($categories);
}
```

**Comment by Gummibeer** [comment]
And this executes `toArray()` on your model. ;)
`json` calls `toJson` on the model which calls `toArray`.
You should fix this by adding a `pluck('id')`
```php
public function simpleCategories()
{
    $categoryIds = Category::select('id')->get()->pluck('id');
    return response()->json($categoryIds);
}
```

**Comment by rafailml** [comment]
Thanks @Gummibeer , that works great, but how can I get that result in more complex query as in my first post?
```
$product = Product::select('id', 'sku',...)
->with(['categories' => function ($query) {
                        $query->select(['categories.id']);
                    }])
->where('id', $id)
->first();

return response()->json($product);
```

**Comment by Gummibeer** [comment]
You could try to do it by https://laravel.com/docs/5.6/collections#method-only or play around with the other collection methods.

But I would tinker with the `toArrayAlwaysLoadsTranslations` or `attributesToArray` method. My idea is to compare available attributes with all fillables or check if eloquent has a property to detect a not complete database record.
I don't have a ready to use solution but would be happy with a PR to add this feature, in best case with a new config.

The 100% best solution for me is a `makeTranslationsHidden` and `makeTranslationsVisible` method like https://laravel.com/docs/5.6/eloquent-serialization#hiding-attributes-from-json that changes a property on the model which is used in the `attributesToArray` method.

But this method has to be called on every model instance - not rly nice for your problem. But for an API I would go for the way to disable translations by default and enable/load them if needed via eloquent eager loading.

**Comment by rafailml** [comment]
Thanks @Gummibeer !

**Comment by Gummibeer** [comment]
I've added a fix in #484 - can you take a look if it solves your problem?
